packages/server/tests/_util.ts

```diff
@@ -6,23 +6,23 @@ export type CloseFn = () => Promise<void>
 export type TestServerInfo = {
   ctx: AppContext
   url: string
+  db: Database
   close: CloseFn
 }
 
 export const runTestServer = async (opts: {
   dbPostgresSchema: string
 }): Promise<TestServerInfo> => {
   const { dbPostgresSchema } = opts
-  const dbPostgresUrl = process.env.DB_POSTGRES_URL || undefined
-
-  const db =
-    dbPostgresUrl !== undefined
-      ? Database.postgres({
-          url: dbPostgresUrl,
-          schema: dbPostgresSchema,
-        })
-      : Database.memory()
+  const dbPostgresUrl = process.env.DB_POSTGRES_URL
```

Comment:
FWIW, in the golang code we are using `DATABASE_URL` with `postgresql://` prefix for postgresql backend (or `sqlite://`, though you don't do sqlite in this project any more). This is a sort-of-standard way to do it (choice of variable name).

But i'm not sure it is best to do that here b/c you are sticking with existing behavior. And this is just a soft norm, not any real win from one variable name or another.

Reply:
Don't see a huge different here, but I'm down to change it

Bigger different in the PDS code, but we can keep all our impl in sync